test-end-to-end/android-maestro/common/add_new_people.yaml

```diff
@@ -0,0 +1,20 @@
+appId: com.toughra.ustadmobile
```

Comment:
I think the password should be a variable, rather than hardcoded.

Reply:
Done Mike, Thanks